ClinGen/clincoded
# Connects to #1003 kl interpretation data in greenbar

This PR implements requirement to display critical interpretation data in variant-interpretation information (green background) area. Specs are:

* In Evidence View mode
 - Change title to "All Existing Interpretations".
 - List all interpretations, current user's on top.
 - Use pencil icon to replace "Continue Interpretation" button. Locate icon with current user's interpretation data.
 - In each interpretation, display: disease-mode of inheritance, user's name, interpretation status: modified pathogenicity (if status is Provisional), last edited date and time.

* In Variant Interpretation Record mode
 - Change title to "My Interpretation".
 - Display disease term and Orphanet number with a link to Orphanet web page specified to the disease.
 - Display calculated Pathogenicity.
 - Display modified Pathogenicity.
 - Display interpretation status.
 - Display last edited date and time.

**Test Steps**
* Start VCI, enter variant id 54321, expect to see
![screen shot 2016-09-27 at 5 56 13 pm](https://cloud.githubusercontent.com/assets/9206696/18897070/ca4c84d0-84db-11e6-993b-7f1f57f01612.png)

* Click "Interpretation +" button and expect to see
![screen shot 2016-09-27 at 5 58 05 pm](https://cloud.githubusercontent.com/assets/9206696/18897096/ff99d188-84db-11e6-86cd-f1428373c129.png)

* Click "Disease +" button, enter ORPHA15, expect to see disease term and Orphanet number in the area as
![screen shot 2016-09-27 at 5 59 41 pm](https://cloud.githubusercontent.com/assets/9206696/18897137/4959f654-84dc-11e6-89c9-6801147f7956.png)

* Click "Inheritance +" button, select "Autosomal dominant inheritance (HP:0000006)".
* Go to Population tab, select Met at BA1, Save and expect to see `Calculated Pathogenicity: Benign` in the area as
![screen shot 2016-09-27 at 6 05 50 pm](https://cloud.githubusercontent.com/assets/9206696/18897222/15a30fc0-84dd-11e6-9ee0-bbfc9844dbff.png)
* Click "View Summary" button, select "Likely Benign" at `Modified Pathogenicity`, enter a reason, check `Change status to "Provisional Interpretation"` box, click Save and expect to see summary data in the area as:
![screen shot 2016-09-27 at 6 10 03 pm](https://cloud.githubusercontent.com/assets/9206696/18897281/b4f8e734-84dd-11e6-9ea7-78db559e8a62.png)
* Log out
* Log in as second user. Start VCI, enter variant 54321 (same) and expect to see first use's interpretation data as:
![screen shot 2016-09-27 at 6 14 15 pm](https://cloud.githubusercontent.com/assets/9206696/18897333/3d660ef8-84de-11e6-9e34-acc41f4e3b18.png)
* Click "Interpretation +" button, associate disease ORPHA891.
* Go to Predictors/Missense tab, select PP3 to "PP3_Strong", and PP2 to "PP2_Strong", click Save
* Click "New Variant Curation" button, enter variant 54321 (same) and expect to see both users interpretations are listed and a pencil icon with current user's as:
![screen shot 2016-09-27 at 6 22 52 pm](https://cloud.githubusercontent.com/assets/9206696/18897463/7a48c76a-84df-11e6-96bb-c605d396442c.png)
* Click the icon, expect to go to `Variant Interpretation Record` mode.
![screen shot 2016-09-27 at 6 26 04 pm](https://cloud.githubusercontent.com/assets/9206696/18897520/e17e12a0-84df-11e6-8422-38ce5ba976ac.png)

**End of Test**
